hoomd/hpmc/integrate.py

```diff
@@ -851,9 +856,12 @@ def format_param_pos(self, param):
         return 'sphere {0}'.format(d);
 
     def get_type_shapes(self):
-        """ Get all the types of shapes in the current simulation
+        """Get all the types of shapes in the current simulation.
+
+        Currently assumes that all 3D shapes are convex.
+
         Returns:
-            A list of dictionaries, one for each particle type in the system. Currently assumes that all 3D shapes are convex.
+            A list of dictionaries, one for each particle type in the system.
```

Comment:
Would it be helpful for the method to document the contents of the dictionary?

Reply:
Perhaps the right thing to do here is to document a well-defined schema for the `type_shapes` JSON format. It is currently used as a de-facto standard by a few different tools in our group.